README.md

```diff
@@ -24,6 +25,48 @@ An operator can dynamically update a Node's role by promoting a Worker to Manage
 
 *Tasks* are organized in *Services*. A service is a higher level abstraction that allows the user to declare the desired state of a group of tasks.  Services define what type of task should be created as well as how to execute them (e.g. run this many instances at all times) and how to update them (e.g. rolling updates).
 
+## Features
+
+Some of *SwarmKit*'s main features are:
+
+- **Orchestration**
+  - **Desired State Reconciliation**: *SwarmKit* will constantly compare the desired state against the current cluster state
```

Comment:
Avoid future: SwarmKit constantly compares...and reconciles.... SwarmKit reschedules